File: service/src/main/java/no/nav/melosys/service/vedtak/publisering/dto/Adresse.java
package no.nav.melosys.service.vedtak.publisering.dto;

import no.nav.melosys.domain.eessi.sed.Adressetype;

public record Adresse(Adressetype adressetype, String gatenavn, String gatenummer, String postnummer, String poststed) {
}

File: domain/src/main/kotlin/no/nav/melosys/domain/eessi/melding/Adresse.kt
package no.nav.melosys.domain.eessi.melding


data class Adresse(
    var by: String = "",
    var bygning: String? = null,
    var gate: String? = null,
    var land: String? = null,
    var postnummer: String? = null,
    var region: String? = null,
    var type: String? = null
)

